packages/components/src/components/Flyout/Flyout.stories.tsx

```diff
@@ -0,0 +1,193 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { Button } from '../Button/Button';
+
+import { Flyout, FlyoutContent, FlyoutRoot } from './Flyout';
+
+type Story = StoryObj<typeof meta>;
+const meta: Meta<typeof FlyoutContent> = {
+    component: FlyoutContent,
+    subcomponents: {
+        // @ts-expect-error - Storybook has the wrong type for TooltipContent
+        'Flyout.Root': FlyoutRoot,
+    },
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    args: {
+        children: 'Hello World',
+    },
+    render: ({ ...args }) => {
+        return (
+            <Flyout.Root>
+                <Flyout.Trigger>
+                    <Button>Click Me</Button>
```

Comment:
Same in all other stories please
```suggestion
                    <Button>Click me</Button>
```

Reply:
changed